mempool/src/pool/store.rs

```diff
@@ -318,15 +321,23 @@ impl MempoolStore {
     }
 }
 
+impl Idable for TxMempoolEntry {
```

Comment:
This is a bad idea. I get the motivation for this, but `TxMempoolEntry` is not "Ideable". Feel free to create a method in it `get_tx_Id()`.